orchagent/portsorch.cpp

```diff
@@ -1575,6 +1581,24 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 {
                     an = (int)stoul(fvValue(i));
                 }
+
+                /* Set port serdes Pre-emphasis */
+                if (fvField(i) == "preemphasis")
+                {
+                    sai_get_port_serdes_values(fvValue(i), pre_emphasis);
```

Comment:
>sai_get_port_serdes_values [](start = 20, length = 26)

app db value do not use json format. Please propose a text format and use that in the app db.

for example: 

value,value

Reply:
please add the proposed scheme in here.

[https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md](https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md)


---
In reply to: [282750165](https://github.com/Azure/sonic-swss/pull/821#discussion_r282750165) [](ancestors = 282750165)